src/openrct2/scripting/bindings/world/ScResearch.hpp

```diff
@@ -0,0 +1,293 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2023 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
```

Comment:
Should we perhaps make this a .cpp file with a header. This is bringing quite a lot of headers in.